src/pages/api/simulation.ts

```diff
@@ -0,0 +1,35 @@
+import { TENDERLY_ACCESS_KEY, TENDERLY_PROJECT, TENDERLY_USER } from '../../consts/config';
+import { failureResult, successResult } from '../../features/api/utils';
+
+export default async function handler(req, res) {
+  const data = req.body;
+  if (!TENDERLY_ACCESS_KEY || !TENDERLY_PROJECT || !TENDERLY_USER) {
+    console.log('ENV not defined');
```

Comment:
Use logger instead of console, I believe this will raise a linter warning

Reply:
Change log message to something more descriptive: `Tenderly key, project, or user not defined in env`